WooCommerce/Classes/ViewRelated/Coupons/CouponDetails/CouponDetails.swift

```diff
@@ -379,6 +391,26 @@ private extension CouponDetails {
             "%1$d uses per user",
             comment: "The plural limit of time for each user to apply a coupon, reads like: 10 uses per user"
         )
+        static let singularItemsInCartUsageLimit = NSLocalizedString(
+                "Limited to %1$d item in cart",
+                comment: "The singular limit of the same item in cart to apply a coupon, reads like: " +
```

Comment:
Nit: maybe we can reword it to make it sound clearer to translators:

```suggestion
                comment: "The singular form of the required number of items in the cart to apply a coupon, reads like: " +
```

Reply:
done here: https://github.com/woocommerce/woocommerce-ios/pull/6831/commits/65e6913a5a970f5a1edbc6b2ab77a7b0e15e19a5